third_party/txt/src/txt/font_collection.cc

```diff
@@ -163,11 +163,16 @@ FontCollection::GetMinikinFontCollectionForFamilies(
     return nullptr;
   }
   if (enable_font_fallback_) {
-    for (std::string fallback_family : fallback_fonts_for_locale_[locale]) {
+    // Reverse iterate to prevent new fallback fonts changing how previously
+    // resolved glyphs are rendered.
+    auto fallback_riter = fallback_fonts_for_locale_[locale].rbegin();
```

Comment:
Change `fallback_fonts_for_locale_` to hold vectors in order to preserve the order in which the fallback fonts were added